packages/grid/x-data-grid/src/models/api/gridLocaleTextApi.ts

```diff
@@ -81,11 +81,15 @@ export interface GridLocaleText {
   // Column menu text
   columnMenuLabel: string;
   columnMenuShowColumns: React.ReactNode;
+  columnMenuManageColumns: React.ReactNode;
   columnMenuFilter: React.ReactNode;
   columnMenuHideColumn: React.ReactNode;
+  columnMenuHideColumnDefault: React.ReactNode;
   columnMenuUnsort: React.ReactNode;
   columnMenuSortAsc: React.ReactNode;
   columnMenuSortDesc: React.ReactNode;
+  columnMenuSortDefaultAsc: React.ReactNode;
+  columnMenuSortDefaultDesc: React.ReactNode;
```

Comment:
That sounds good, by the way, do we want to deprecate the simple one ultimately? I feel keeping it would be better to allow users an additional design to use if they want.